bundle/deploy/terraform/init.go

```diff
@@ -39,6 +38,21 @@ func (m *initialize) findExecPath(ctx context.Context, b *bundle.Bundle, tf *con
 		return tf.ExecPath, nil
 	}
 
+	// Load exec path from the environment if it matches the currently used version.
+	envTFVersion := env.Get(ctx, "DATABRICKS_TF_VERSION")
+	envExecPath := env.Get(ctx, "DATABRICKS_TF_EXEC_PATH")
+	if envExecPath != "" && envTFVersion == TerraformVersion.String() {
+		_, err := os.Stat(envExecPath)
+		if err != nil && !os.IsNotExist(err) {
+			return "", err
+		}
+		if err == nil {
+			tf.ExecPath = envExecPath
+			log.Debugf(ctx, "Using Terraform from the environment at %s", tf.ExecPath)
+			return tf.ExecPath, nil
+		}
+	}
```

Comment:
If this is specified but some error happened, we should provide clear error messages.

For example, if you set `DATABRICKS_TF_EXEC_PATH` and it doesn't exist, then we get a file not found error.

Reply:
I've added more debug logs for the error cases. In most of them we don't error-out though, just ignore the env variables.